EXPLAINER_fullscreen_popups.md

```diff
@@ -0,0 +1,161 @@
+# Multi-Screen Window Placement on the Web - Creating Fullscreen Popup Windows
+
+## Introduction
+This proposal aims to reduce user and developer friction for initiating a fullscreen experience on the desired display of a multi-screen device. The proposed web platform enhancement allows web applications to open a new window in HTML [fullscreen](https://fullscreen.spec.whatwg.org/) mode on a specific display with a single user gesture.
+
+## Background
+[Multi-Screen Window Placement](https://github.com/w3c/window-placement) is a new permission-gated web platform API that provides web applications with information about the device's connected screens, allowing them to open and position windows or request fullscreen on any of those screens.
+
+The HTML living standard specifies [Tracking User Activation](https://html.spec.whatwg.org/multipage/interaction.html#tracking-user-activation), to gate some web platform APIs on user interaction signals. 
+
+## Problem

```

Comment:
This could mention that web apps wish to show fullscreen content on a specific display, alongside the original content, but target-display fullscreen makes the existing window otherwise inaccessible. (or similar)

Reply:
Added, with the addition of "may make the existing window inaccessible" since in-theory other browsers like Safari would still allow access to the original window.